src/NuGet.Core/NuGet.CommandLine.XPlat/Program.cs

```diff
@@ -97,6 +97,7 @@ public static int MainInternal(string[] args, CommandOutputLogger log, IEnvironm
                 {
                     var packageCommand = new Command("package");
                     rootCommand.Subcommands.Add(packageCommand);
+                    PackageDownloadCommand.Register(packageCommand, getHidePrefixLogger);
```

Comment:
The modern way for to get NuGet commands into the .NET SDK is via [NuGetCommands.cs](https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.CommandLine.XPlat/NuGetCommands.cs).  So, you should also register it there.

Reply:
It should be in both Program.cs and NuGetCommands.cs.  Program.cs makes it easy for us to debug.